assets/queries/cloudFormation/iot_policy_allows_action/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "iot_policy_allows_action",
```

Comment:
Change **directory name** to: `iot_policy_allows_wildcard_action`
Change **id** to: `iot_policy_allows_wildcard_action`
Change **query name** to: `IoT Policy Allows Action as Wildcard`